_articles/building-the-best-one-click-sign-in.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: article
+title: "Building The Best One Click Sign In"
+abstract: "One click sign in using device authentication offers convenience and security to the user."
+sitemap:
+  lastmod: 2020-05-18
+---
+
+One click sign in allows users to authenticate in just one click.  Delivering this sought after feature was a challenge but we are pleased with the results. This is our story of implementing one click sign in with DID.app.
+
+## Why do we need one click sign in?
+
+Users have ‘account creation fatigue’. The average person already has 150+ accounts and the pace of growth shows no sign of slowing. There are so many websites that want you to create an account and that's why we need one click sign in.
+
+Users want to engage without filling in endless forms, choosing yet another password or confirming their email address.  Users want convenience.  Or, to put it another way, users are lazy; I know I am.
+
+## How does one click sign in work?
+
+Single factor authentication can relies on either: something you know, something you have or something you are.  In a typical username and password setup, the user relies on something only they know (their password) to prove their identity.  For one click sign in, we rely on something the user has.  The one thing every user has is a device. 
```

Comment:
```suggestion
Single factor authentication can relies on either: [something you know, something you have or something you are](/articles/the-three-factors-of-authentication).  In a typical username and password setup, the user relies on something only they know (their password) to prove their identity.  For one click sign in, we rely on something nearly ever user already has with them, their mobile phone. 
```

Reply:
Well, yes, I like to use my laptop too.  The flow on the phone isn't as good as it is on my laptop.  I secure my laptop with a pin code.  Also, mobile phones can include Nokie 42 tens or whatever they are called.    I'd prefer to go with 'device'.

Reply:
Updated to say their personal devices